package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "starknetkit",
-  "version": "1.1.3",
+  "version": "2.0.0",
```

Comment:
are we sure we want to jump to 2.0.0?

Reply:
I think it makes sense since they're breaking changes

Reply:
It's technically a breaking change to the StarknetWindowObject. So yes, I think

Reply:
Using `semantic-release`, we should not change the version manually but reflect the breaking change in a commit message that will trigger a major release bump. see https://semantic-release.gitbook.io/semantic-release/